types/Yargs.ts

```diff
@@ -0,0 +1,10 @@
+import { Arguments } from 'yargs';
+
+export type CommonOptions<T = object> = Arguments<
```

Comment:
Since we decided to use camel case, how would you feel about `ArgumentsCamelCase`?  Example https://github.com/HubSpot/hubspot-cli/blob/main/commands/doctor.ts#L26C4-L26C22

Reply:
Switched to this in [001438e](https://github.com/HubSpot/hubspot-cli/pull/1342/commits/001438e543c4042a85fb71987e3e596a2b2db25d)